client/state/jetpack-remote-install/reducer.js

```diff
@@ -0,0 +1,43 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { createReducer, combineReducers, keyedReducer } from 'state/utils';
+import {
+	JETPACK_REMOTE_INSTALL,
+	JETPACK_REMOTE_INSTALL_FAILURE,
+	JETPACK_REMOTE_INSTALL_SUCCESS,
+} from 'state/action-types';
+
+export const isComplete = keyedReducer(
+	'url',
+	createReducer( false, {
+		[ JETPACK_REMOTE_INSTALL_SUCCESS ]: () => true,
+		[ JETPACK_REMOTE_INSTALL ]: () => false,
+	} )
+);
+
+export const inProgress = keyedReducer(
+	'url',
+	createReducer( false, {
+		[ JETPACK_REMOTE_INSTALL ]: () => true,
+		[ JETPACK_REMOTE_INSTALL_SUCCESS ]: () => false,
+		[ JETPACK_REMOTE_INSTALL_FAILURE ]: () => false,
+	} )
+);
+
+export const error = keyedReducer(
+	'url',
+	createReducer( '', {
```

Comment:
Is there motivation for `''` as opposed to `null`?

Reply:
no, will change...